libs/subliminal_patch/providers/subf2m.py

```diff
@@ -82,12 +85,36 @@ def id(self):
     "dutch": "dut",
     "hebrew": "heb",
     "indonesian": "ind",
+    "danish": "dan",
+    "norwegian": "nor",
+    "bengali": "ben",
+    "bulgarian": "bul",
+    "croatian": "hrv",
+    "swedish": "swe",
+    "vietnamese": "vie",
+    "czech": "cze",
+    "finnish": "fin",
+    "french": "fre",
+    "german": "ger",
+    "greek": "gre",
+    "hungarian": "hun",
+    "icelandic": "ice",
+    "japanese": "jpn",
+    "macedonian": "mac",
+    "malay": "may",
+    "polish": "pol",
+    "romanian": "rum",
+    "russian": "rus",
```

Comment:
You can add `Serbian` as well being that it is supported on `subf2m`
`"serbian": "srp",`